Adding new use case for DD_KUBERNETES_POD_LABELS_AS_TAGS (#5028)
DataDog/documentation: content/en/agent/autodiscovery/tag.md

  app: kube_app
```

> Use the following environment variable configuration to add all pod labels as tags to your metrics. In this example the tags names are prefixed by `<PREFIX>_`:

Reviewer: ```suggestion
For Agent v6.8.0+, use the following environment variable configuration to add all pod labels as tags to your metrics. In this example, the tag names are prefixed by `<PREFIX>_`:
```

```yaml
kubernetes_pod_labels_as_tags:
  *: <PREFIX>_%%label%%
```

> **Note**: Doing so may infinitely [increase the number of metrics][2] for your organization and impact your billing.

Reviewer: ```suggestion
**Note**: Using this method may [increase the number of metrics][2] for your organization and impact your billing.
```

DD_KUBERNETES_POD_LABELS_AS_TAGS='{"app":"kube_app"}'
```

> Use the following environment variable configuration to add all pod labels as tags to your metrics. In this example the tags names are prefixed by `<PREFIX>_`:

Reviewer: ```suggestion
For Agent v6.8.0+, use the following environment variable configuration to add all pod labels as tags to your metrics. In this example, the tags names are prefixed by `<PREFIX>_`:
```

```shell
DD_KUBERNETES_POD_LABELS_AS_TAGS='{"*":"<PREFIX>_%%label%%"}'
```

> **Note**: Doing so may infinitely [increase the number of metrics][1] for your organization and impact your billing.

Reviewer: ```suggestion
**Note**: Using this method may [increase the number of metrics][2] for your organization and impact your billing.
```
